akka-actor-typed-tests/src/test/scala/akka/actor/typed/scaladsl/ScalaDslActorContextSpec.scala

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright (C) 2009-2018 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.actor.typed.scaladsl
+
+import akka.actor.typed.{ ActorRef, TypedAkkaSpec }
+import akka.testkit.typed.TestKit
+import akka.testkit.typed.scaladsl.TestProbe
+
+import scala.util.{ Failure, Success }
+
+class ScalaDslActorContextSpec extends TestKit with TypedAkkaSpec {
```

Comment:
I think it's better to have more specific test classes, like ScalaDslActorContextAskSpe, rather than one that encourage testing everything of ActorContext in same file.

Reply:
I agree, actually already renamed it like that after pushing this